test/unit/test_s_expr.cpp

```diff
@@ -710,35 +710,15 @@ std::string round_trip_component(std::istream& stream) {
     }
 }
 
-TEST(decor_literals, double_to_iexpr_promotion) {
```

Comment:
The s-exp constructor of `membrane-potential` et al has changed, since the type has changed.
Before, we used 
```cxx
struct membrane_potential {
  iexpr value;
};
```
which has turned out to be a less than ideal design. Now, we have
```cxx
struct membrane_potential {
  double value;
  iexpr scale = 1;
};
```
We could still promote from doubles, **but** that'll clash with the old format. So, I decided
to make this explicit and bump the format. Something that has been forgotten a few times
it seems, as we still use `acc-format: 0.1-dev`.